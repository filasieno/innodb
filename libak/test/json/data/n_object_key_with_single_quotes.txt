name=n_object_key_with_single_quotes
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
{key: 'value'}