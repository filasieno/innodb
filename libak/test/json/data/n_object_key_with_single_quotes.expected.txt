---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_OBJECT
STATE_CHANGED_EVENT: STATE_ERROR 223
